hapi-fees/src/main/java/com/hedera/services/usage/token/TokenWipeUsage.java

```diff
@@ -45,7 +45,7 @@ public FeeData get() {
 		addEntityBpt();
 		addAmountBpt();
 		addEntityBpt();
-		addTokenTransfersRecordRb(1, 1);
+		addTokenTransfersRecordRb(1, 1, 1);
```

Comment:
We are not adding nft transfer records on TokenWipe (for now). It must be `1,1,0`

@joan41868 this must be updated with your Wipe implementation

Reply:
Corrected!